src/app.js

```diff
@@ -0,0 +1,29 @@
+import './assets/scss/app.scss';
+import $ from 'cash-dom';
+
+
+export class App {
+  initializeApp() {
+    let self = this;
+
+    $('.load-username').on('click', function (e) {
```

Comment:
Since we are using `class` keyword which is from es6+, then we should also change `function(e)` to arrow function like `(e) =>` and delete variable `self` and replace it with `this`, since arrow function does not creating new `this` scope.